src/main/java/org/mitre/synthea/export/BB2RIFExporter.java

```diff
@@ -2600,13 +2601,17 @@ private long exportHome(Person person, long startTime, long stopTime) throws IOE
     long maxGapForContinuousHHAService = Utilities.convertTime("days", 2);
     ConsolidatedServicePeriods servicePeriods = new ConsolidatedServicePeriods(
             maxGapForContinuousHHAService);
-    for (HealthRecord.Encounter encounter : person.record.encounters) {
+    for (Encounter encounter : person.record.encounters) {
       if (encounter.stop < startTime || encounter.stop < claimCutoff) {
         continue;
       }
       if (!getClaimTypes(encounter).contains(ClaimType.HHA)) {
         continue;
       }
+      List<String> mappedDiagnosisCodes = getDiagnosesCodes(person, encounter.stop);
+      if (mappedDiagnosisCodes.isEmpty()) {
+        continue; // skip this encounter
+      }
```

Comment:
Should we really be doing this when the diagnosis codes are optional?

To be solved later....